lang/python/python-tornado/Makefile

```diff
@@ -0,0 +1,54 @@
+#
+# Copyright (C) 2020 CZ.NIC z.s.p.o. (https://www.nic.cz/)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python-tornado
+PKG_VERSION:=6.0.4
+PKG_RELEASE:=1
+
+PYPI_NAME:=tornado
+PKG_HASH:=0fe2d45ba43b00a41cd73f8be321a44936dc1aba233dee979f17a042b83eb6dc
+
+PKG_MAINTAINER:=Josef Schlehofer <josef.schlehofer@nic.cz>
+PKG_LICENSE:=Apache-2.0
+PKG_LICENSE_FILES:=LICENSE
+
+include ../pypi.mk
+include $(INCLUDE_DIR)/package.mk
+include ../python3-package.mk
+
+define Package/python3-tornado
+  SECTION:=lang
+  CATEGORY:=Languages
+  SUBMENU:=Python
+  TITLE:=Web framework and asynchronous networking library
+  URL:=https://tornadoweb.org
+  DEPENDS:= \
+    +python3-asyncio \
+    +python3-codecs \
+    +python3-ctypes \
```

Comment:
ctypes appears to only be imported in [tornado/platform/windows.py](https://github.com/tornadoweb/tornado/blob/v6.0.4/tornado/platform/windows.py#L4-L5), which I believe is only used when the OS is Windows, so I don't think `+python3-ctypes` is necessary.

Reply:
🤦 You are correct.